MapboxDirectionsTests/V5Tests.swift

```diff
@@ -58,6 +58,7 @@ class V5Tests: XCTestCase {
         XCTAssertEqual(route!.accessToken, BogusToken)
         XCTAssertEqual(route!.apiEndpoint, URL(string: "https://api.mapbox.com"))
         XCTAssertEqual(route!.routeIdentifier, "cj725hpi30yp2ztm2ehbcipmh")
+        XCTAssertEqual(route!.spokenLocale!.description, "en-US")
```

Comment:
`description` is a free-form, human-readable string that may contain extra details beyond the identifier. Use `identifier` instead.